frontend/src/scenes/settings/SettingsMap.tsx

```diff
@@ -108,7 +108,7 @@ export const SETTINGS_MAP: SettingSection[] = [
                 id: 'exception-autocapture',
                 title: 'Exception autocapture',
                 component: <ExceptionAutocaptureSettings />,
-                flag: 'EXCEPTION_AUTOCAPTURE',
+                flag: 'ERROR_TRACKING',
```

Comment:
I assume you've copied over release conditions for exception-autocapture into error tracking? So users already enabled for this wouldn't suddenly lose access

Reply:
Yep, it was only enabled for the former Replay team